pkg/controller/node/nodecontroller.go

```diff
@@ -32,11 +31,13 @@ import (
 	"k8s.io/apimachinery/pkg/types"
 	utilruntime "k8s.io/apimachinery/pkg/util/runtime"
 	"k8s.io/apimachinery/pkg/util/wait"
+
```

Comment:
I wouldn't object if we followed the style here throughout the code base. Much easier to read instead of 30 apimachinery+client-go+apiserver+kube imports without structure.